css/header.css

```diff
@@ -125,26 +138,26 @@ header a {
   justify-content: space-between;
   align-items: center;
   font-size: 2rem;
-  padding: 2.4rem 3.8rem 2.4rem 4.6rem;
+  padding: 1.2em 1.9em 1.2em 2.3em;
```

Comment:
from rem you are switching back to em, and in line 140 you are using rem. what happened?